packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1527,6 +1529,19 @@ class StackViewport extends Viewport implements IStackViewport {
     return imageId;
   }
 
+  /**
+   * Throws an error if you are using a destroyed instance of the stack viewport
+   */
+  private _throwIfDestroyed() {
+    if (this.isDisabled) {
+      throw new Error(
+        'The stack viewport has been destroyed and is no longer usable. Renderings will not be performed. If you ' +
+          'are using the same viewportId and has re-enabled the viewport, you need to grab the new viewport instance ' +
+          'using renderingEngine.getViewport(viewportId), instead of using your lexical reference to the viewport instance.'
```

Comment:
What do you mean by lexcial here? Wouldn't 'previous' or 'existing' be better words to use?

Reply:
I mean lexical scope https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures
sure i will add those too

Reply:
Right. Sorry. Then say "lexical scoped reference"